### PR TITLE
Bug 1430720 Added more info to F5 docs

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1430720

And: #3774 

@rajatchopra PTAL

### CODE REVIEW

**Review by rajatchopra — APPROVED** [review]
Looks good to me

**Comment by bfallonf** [comment]
Great. Thanks @rajatchopra . I'll move this along.

@openshift/team-documentation PTAL

File: install_config/router/f5_router.adoc [hunk 1]
```diff
@@ -27,19 +27,41 @@ xref:../../install_config/router/default_haproxy_router.adoc#install-config-rout
...
 When deploying the F5 router plug-in, ensure you meet the following
 requirements:
 
-. An F5 host IP and credentials
-. The name of the virtual servers (for both HTTP and HTTPS)
-. The private key to the F5 instance
-. The host-internal IP and VXLAN gateway
+* A F5 host IP with:
+** Credentials for API access
+** ssh access via a private key
```
Comment (mburke5678) [inline install_config/router/f5_router.adoc]: it appears we use all caps for SSH when not used as a command:
s/ssh access/SSH access

File: install_config/router/f5_router.adoc [hunk 2]
```diff
@@ -27,19 +27,41 @@ xref:../../install_config/router/default_haproxy_router.adoc#install-config-rout
...
-. Ensure you have xref:../../install_config/router/index.adoc#creating-the-router-service-account[created the router service account].
+* Ensure you have xref:../../install_config/router/index.adoc#creating-the-router-service-account[created the router service account].
 endif::[]
 
+{product-title} supports only the following *F5 BIG-IP®* versions:
+
+* 11.x
+* 12.x
+
+Adding to that, the following features are not supported:
```
Comment (mburke5678) [inline install_config/router/f5_router.adoc]: Do we need _Adding to that,_?

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/router/f5_router.adoc#install-config-router-f5[Setting up a Router -> Using the F5 Router Plug-in]
|Added to the xref:../install_config/router/f5_router.adoc#install-config-router-f5#install-router-f5-prerequisites[Prerequisites and Supportability] section with supportability information.
%

**Comment by bfallonf** [comment]
Thanks @mburke5678 . I'll merge.

**Comment by vikram-redhat** [comment]
@bfallonf - the link to the prereq section is not correct. Can you please fix.

**Comment by bfallonf** [comment]
@vikram-redhat done

**Comment by vikram-redhat** [comment]
@bfallonf - I don't think these changes applied to 3.4 and 3.5 docs because they include information about virtual servers, which was included only in 3.6 (https://github.com/openshift/openshift-docs/pull/4534). At the moment, I have applied this to 3.5, but not 3.4. Can you please confirm and either remove this from 3.5, or add it to 3.4 if applicable?

**Comment by bfallonf** [comment]
Followup: #5297 put this into 3.4.

**Pull request merged**